### PR TITLE
drivers/segger: Add Segger RTT channel initialization

### PR DESCRIPTION
**Pull request by Gary-Hobson**
## Summary
Add Segger RTT channel initialization

### CODE REVIEW

**Comment by acassis** [comment]
@Gary-Hobson could you please include a Documentation/ about it?

**Comment by Gary-Hobson** [comment]
> @Gary-Hobson could you please include a Documentation/ about it?

Of course, I will submit a separate readme about the use of Segger J Link RTT later

Or submit it I use defconfig which is based on stm32 743ZI nucleo board

File: drivers/segger/rtt_initialize.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: rtt_channel_initialize
+ ****************************************************************************/
+
+void rtt_channel_initialize(int channel, FAR char * name,
```
Comment (xiaoxiang781216) [inline drivers/segger/rtt_initialize.c:41]: why not do the initialize in lib_rtt[out|in]stream_init?

Comment (Gary-Hobson) [reply drivers/segger/rtt_initialize.c:41]: Initializing at system startup is a cleaner way

lib_rttoutstream_init may be called multiple times, which leads to the need to determine whether it has been initialized

Moreover, it is possible to write data through rtt_serial, but lib_rttoutstream_init is not called for initialization

**Pull request closed without merging**